.shippable.yml

```diff
@@ -25,6 +25,7 @@ build:
         options: "-e HOME=/home/buildslave --privileged=true --tty --net=bridge --user buildslave"
 
     ci:
+      - sudo pip3 install psutil
```

Comment:
I think you should be able to remove this, it is already part of the docker..

Reply:
@nashif update